From be859414ba41001747b03b1066e2a198886041db Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 31 Mar 2014 13:59:34 -0400
Subject: [PATCH 14/41] Fix bad check for new-kernel-pkg's command line
 arguments.

--devtree's argument needs to be nonempty, not just pass -f.
Resolves: rhbz#1082318

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 new-kernel-pkg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/new-kernel-pkg b/new-kernel-pkg
index 97e1839..034fa3b 100755
--- a/new-kernel-pkg
+++ b/new-kernel-pkg
@@ -153,7 +153,7 @@ install() {
     fi
 
     DEVTREE=""
-    if [ -f $devtreefile ]; then
+    if [ "x$devtreefile" != "x" -a -f $devtreefile ]; then
 	[ -n "$verbose" ] && echo "found $devtreefile and using it with grubby"
 	DEVTREE="--devtree $devtreefile"
     fi
-- 
2.4.3

